rules/no-unnecessary-polyfills.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+const semver = require('semver');
+const readPkgUp = require('read-pkg-up');
+const builtIns = require('@babel/preset-env/data/built-ins');
```

Comment:
It's obsolete data, use `core-js-compat`.

Reply:
I could not find node versions in `core-js-compat`

Reply:
@doniyor2109 what do you mean? https://www.npmjs.com/package/core-js-compat